packages/lesswrong/components/messaging/ConversationContents.tsx

```diff
@@ -17,15 +16,17 @@ const styles = (theme: ThemeType): JssStyles => ({
     marginBottom: 12,
   },
   editor: {
-    margin: '32px 0px',
     position: "relative",
     '& .form-submit': {
       // form-submit has display: block by default, which for some reason makes it take up 0 height
       // on mobile. This fixes that.
       display: "flex",
     },
-    ...(isEAForum && {
-      borderTop: theme.palette.border.grey200,
+    ...(isEAForum ? {
```

Comment:
I think all the changes in this file should probably be `isFriendlyUI`, no?

Reply:
I see `const InboxComponent = isFriendlyUI ? FriendlyInbox : InboxNavigation;` in `InboxWrapper.tsx` - doesn't that mean that WU are using the new DM page too?